### DISCUSSION TITLE
Targetting browsers that do not support native ESM or recent Ecmascript features

### DISCUSSION THREAD

**Discussion by jeffcunat** [nuxt/nuxt#15965] [Questions] [answered]
Is it possible to target legacy browsers (for example 2018 Samsung or LG SmartTV browsers that do not support ESM or other Ecmascript improvements like spread operators, nullish coalescing operator)
Note that Tv browsers are not able to be updated...

Browser references:
https://developer.samsung.com/smarttv/develop/specifications/web-engine-specifications.html
https://webostv.developer.lge.com/discover/specifications/web-engine/
 
I have tried with Vite and vite plugin legacy https://github.com/vitejs/vite/tree/main/packages/plugin-legacy
And also with webpack with custom configuration without success

Any ideas ?

**Comment by jeffcunat** [comment, marked as answer]
I have successfully run a code on a Chrome 55 tv browser with this webpack config:

````
import { defineNuxtConfig } from 'nuxt'
import { ESBuildMinifyPlugin } from 'esbuild-loader' 
// https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
  builder: 'webpack',
  app: {
    head: {
      script: [
        // polyfill to get es2015 features and globalThis
        { src: 'https://polyfill.io/v3/polyfill.min.js?features=es2015%2CglobalThis' }
      ]
    }
  },
  hooks: {
    'webpack:config': (configs) => {
      // configs[0] is the config for the client side
      configs[0].optimization.minimizer = [
        new ESBuildMinifyPlugin({
          target: 'es2015'
        })
      ]
    }
  }
})
````

**Reply by jeffcunat** [reply]
This does not work anymore in rc.8 because all scripts are now considered as ESM and are imported with <script type="module"> syntax
I have proposed a PR to workaround that : https://github.com/nuxt-contrib/vue-bundle-renderer/pull/38

**Reply by rafifn** [reply]
hey @jeffcunat, do you have a solution for `v3.0.0`?
